.gitignore

```diff
@@ -1,4 +1,3 @@
 /coverage/
 *.gemfile.lock
-/Gemfile.lock
```

Comment:
I've gone back and forth on this, but ultimately [follow the docs](https://bundler.io/guides/faq.html) now.

> Q: Should I commit my Gemfile.lock when writing a gem?
> A: Yes, you should commit it.